src/Modules/OrchardCore.Commerce.Tax/Views/TaxRateSettings_Edit.cshtml

```diff
@@ -0,0 +1,147 @@
+    <label asp-for="SourceCity">@T["City"]</label>
+    <input asp-for="SourceCity" class="form-control" type="text"/>
+    <span asp-validation-for="SourceCity"></span>
+</div>
+
+<div class="mb-3" asp-validation-class-for="SourceProvince">
+    <label asp-for="SourceProvince">@T["State or province code"]</label>
+    <input asp-for="SourceProvince" class="form-control" type="text"/>
+    <span asp-validation-for="SourceProvince"></span>
+</div>
+
+<div class="mb-3" asp-validation-class-for="SourcePostalCode">
+    <label asp-for="SourcePostalCode">@T["Postal code"]</label>
+    <input asp-for="SourcePostalCode" class="form-control" type="text"/>
+    <span asp-validation-for="SourcePostalCode"></span>
+</div>
+
+<div class="mb-3" asp-validation-class-for="SourceRegion">
+    <label asp-for="SourceRegion">@T["Country or region"]</label>
+    <input asp-for="SourceRegion" class="form-control" type="text"/>
+    <span asp-validation-for="SourceRegion"></span>
+</div>
+
+<h2>@T["Recipients and Tax Codes"]</h2>
+
+<table id="@tableElementId" class="table @tableElement">
+    <thead>
+    <tr>
+        <th>@T["First street address"]</th>
+        <th>@T["Second street address"]</th>
+        <th>@T["City"]</th>
+        <th>@T["State or province code"]</th>
+        <th>@T["Postal code"]</th>
+        <th>@T["Country or region code"]</th>
+        <th>@T["Tax Code"]</th>
+        <th>@T["Tax Rate (%)"]</th>
+        <th>@T["Actions"]</th>
+    </tr>
+    </thead>
+
+    <tbody>
+    <tr v-for="(rate, index) in rates">
+        <td>
+            <input :name="'ISite.Rates[' + index + '].DestinationStreetAddress1'" v-model="rate.destinationStreetAddress1">
+        </td>
+        <td>
+            <input :name="'ISite.Rates[' + index + '].DestinationStreetAddress2'" v-model="rate.destinationStreetAddress2">
+        </td>
+        <td>
+            <input :name="'ISite.Rates[' + index + '].DestinationCity'" v-model="rate.destinationCity">
+        </td>
+        <td>
+            <input :name="'ISite.Rates[' + index + '].DestinationProvince'" v-model="rate.destinationProvince">
+        </td>
+        <td>
+            <input :name="'ISite.Rates[' + index + '].DestinationPostalCode'" v-model="rate.destinationPostalCode">
+        </td>
+        <td>
+            <input :name="'ISite.Rates[' + index + '].DestinationRegion'" v-model="rate.destinationRegion">
+        </td>
+        <td>
+            <input :name="'ISite.Rates[' + index + '].TaxCode'" v-model="rate.taxCode">
+        </td>
+        <td>
+            <input :name="'ISite.Rates[' + index + '].TaxRate'" v-model="rate.taxRate" type="number" min="0" max="100" step="0.001">
+        </td>
+        <td>
+            <button class="btn btn-danger" @@click.prevent="rates.splice(index, 1)">
+                <i class="fa fa-trash" aria-hidden="true"></i>
+            </button>
+            <button class="btn btn-secondary"
+                    :disabled="index + 1 == rates.length"
```

Comment:
Yes, that's normal, because the IDE thinks it's looking at raw HTML when it's Vue template syntax. We should be able to port this to a .vue file later if we have more use for Lombiq.VueJS. Right now it felt like an overkill to add it just for this very simple dynamic table.